src/main.rs

```diff
@@ -7,8 +7,6 @@ use windows::Win32::{
     UI::WindowsAndMessaging as wm,
 };
 
-#[cfg(test)]
```

Comment:
After updating my Rust environment by a few versions, I found I could no longer compile non-test builds, due to a missing wide_string module. It works again if I delete this line. My understanding is that this would omit the next line from non-test builds, right? But this annotation isn't new. Why did this used to work?

Reply:
Opps. It is a problem with https://github.com/Eh2406/rust-reader/pull/22/files#diff-42cb6807ad74b3e201c5a7ca98b911c5fa08380e942be6e4ac5807f8377f87fc `#[cfg(test)]` and `#[macro_use]` modified the `extern crate lazy_static;` until we removed it. Now it is incorrectly applying to `wide_string`.

Reply:
I'll push a fix.

Reply:
Thanks for the quick fix. Does the rest of this PR (as well as #23) look good to merge?